### DISCUSSION TITLE
Problems running the game

### DISCUSSION THREAD

**Discussion by KLAndorfer** [PhilippvK/playforia-minigolf#97] [General]
Hey, thanks for supporting this game first of all :)
 I got some troubles getting it to work, it would be really cool if you could help me out :)
So what I'm doing is I'm downloading the server.jar, client.jar and tracks.zip from the latest release(v2.1.1.0), now im running the server.jar with java and after that im running client.jar and the game starts, however when I make a solo game I cant see any tracks, it's just green, am I doing something wrong here?
Thanks for helping me out!

**Comment by PhilippvK** [comment]
@KLAndorfer  Hey, I was able to reproduce your problem and figured out how you should be be able to fix it:

1. Make sure to unzip the `tracks.zip` file to the same directory where your `server-2.1.1.0-BETA.jar` is located.
2. Depending on how you have unzipped the archive, it might happen that there is an unwanted nested `tracks` folder e.g. `./tracks/tracks/tracks/` and `./tracks/tracks/sets/` are used instead of `./tracks/tracks/` and `./tracks/sets`. You should be able to fix this by yourself bei either moving the `JAR` inside the first `tracks` directory or removing the unwanted directory level.

Good Luck!

PS: If you run the server JAR on the command line, e.g. using `java -jar server-2.1.1.0-BETA.jar` in windows CMD.exe there will be some logging output including error message which might help understanding where the problem lies.

PS 2: Feel free to use the newer release as it should also work and contains a few fixes... (https://github.com/PhilippvK/playforia-minigolf/releases/tag/v2.1.2.0-BETA)

**Reply by KLAndorfer** [reply]
Thanks a lot, I will try :)
EDIT: Works perfectly fine now, thanks again.
